mobile_back_samsung/cpp/eden_mlperf_c_backend/jni/eden_sdk/include/backend_c.h

```diff
@@ -0,0 +1,148 @@
+/* Copyright 2020 The MLPerf Authors. All Rights Reserved.
```

Comment:
You shouldn't need this file anymore. Its available at android/cpp/c/backend_c.h